src/metabase/api/card.clj

```diff
@@ -362,11 +363,11 @@
   "Compute a 'magic' cache TTL time (in seconds) for QUERY by multipling its historic average execution times by the `query-caching-ttl-ratio`.
    If the TTL is less than a second, this returns `nil` (i.e., the cache should not be utilized.)"
   [query]
-  (when-let [average-duration (qputil/query-average-duration query)]
+  (when-let [average-duration (query/average-execution-time-ms (qputil/query-hash query))]
```

Comment:
this is less an optimization and more a way to be able to do a join in SQL when analyzing that table. Eg, with that link, I would be able to create simple dashboards in sql. without it, I'd need to make that connection via code somehow.